src/main/java/net/pms/dlna/virtual/MediaLibraryFolder.java

```diff
@@ -36,30 +56,38 @@ public void discoverChildren() {
 				if (expectedOutput == FILES) {
 					ArrayList<File> list = database.getFiles(sql);
 					if (list != null) {
-						UMSUtils.sort(list, PMS.getConfiguration().mediaLibrarySort());
+						UMSUtils.sort(list, PMS.getConfiguration().getSortMethod(null));
```

Comment:
@Nadahar Yes, for MediaLibrary class, performance becomes important. A sort field can be added to the DB instead of running FileUtil.renameForSorting() every time. This takes care of all use cases except SORT_LOC_NAT and SORT_RANDOM.